### ISSUE TITLE
POST PUT DELETE not updating json

### ISSUE THREAD

**Issue by troshan** [typicode/jsonplaceholder#19]
None of the write methods actually updating the josn db.

**Comment by Resoguy** [comment]
yes, i am trying to update or add with the angular, it gives no error and actually doing the navigating to another page like it finished succesfully worked but there is no change on the list items on the main page.

I think it is intended that way but i try to console.log the response if it is success or not it returns undefined.

**Comment by FredericoSilva015** [comment]
Please read the documentation, it specifically states, the server does not create a new entry but "fakes it", if you head into the network side of chrome and monitor it, you will see the requests being passed to the server in the xhr tab. So if you have done thing's properly, the server should return you a 200, a failure means a 404.
